purchase_security/security/security.xml

```diff
@@ -8,10 +8,19 @@
                 eval="[(4, ref('purchase.group_purchase_user'))]"
             />
         </record>
+        <record id="group_purchase_group_orders" model="res.groups">
+            <field name="name">User (team orders)</field>
+            <field name="category_id" ref="base.module_category_inventory_purchase" />
+            <field
+                name="implied_ids"
+                eval="[(4, ref('purchase_security.group_purchase_own_orders'))]"
+            />
+        </record>
         <record id="purchase.group_purchase_manager" model="res.groups">
             <field
                 name="implied_ids"
-                eval="[(3, ref('purchase.group_purchase_user')), (4, ref('group_purchase_own_orders'))]"
+                eval="[(3, ref('purchase.group_purchase_user')),
+                (4, ref('group_purchase_own_orders')),(4, ref('group_purchase_group_orders'))]"
```

Comment:
I don't think you need to imply these groups as managers override the access rules